integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -478,6 +479,51 @@ public static boolean doesPodLogContainString(String namespace, String podName,
     return podLog.contains(expectedString);
   }
 
+  /**
```

Comment:
Where is the range parameter as the method name implies ?  Can you give an example on the method description

Reply:
added description